mysql/toddl.go

```diff
@@ -191,3 +193,38 @@ func cvtPrimaryKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Key)
 	}
 	return spKeys
 }
+
+func cvtForeignKeys(conv *internal.Conv, srcTable string, srcKeys []schema.ForeignKey, schemaForeignKeys map[string]bool) []ddl.Foreignkey {
```

Comment:
Just wanted to document in the PR: we haven't included a test in toddl_test.go for foreign keys and that's expected.

Background: toddl_test.go just has just a very basic smoke test of toddl.go.
The main testing of toddl.go code is in infoschema_test.go and mysqldump_test.go.